src/index.html

```diff
@@ -7,6 +7,257 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="navigation">
+      <div class="navigation__wraper">
+        <div class="navigation__logo">
+          <img
+            src="images/logo.png"
+            alt="moyo logo"
+            srcset=""
+          >
+        </div>
+        <nav class="navigation__nav">
+          <ul class="navigation__list">
+            <li class="navigation__item"><a href="#" class="navigation__link is-active">Apple</a></li>
+            <li class="navigation__item"><a href="#" class="navigation__link">Samsung</a></li>
+            <li class="navigation__item"><a href="#" class="navigation__link">Смартфоны и телефоны</a></li>
+            <li class="navigation__item">
+              <a href="#"
+                 class="navigation__link"
+                 data-qa="nav-hover">
+              Ноутбуки и компьютеры
+              </a>
+            </li>
+            <li class="navigation__item"><a href="#" class="navigation__link">Гаджеты</a></li>
+            <li class="navigation__item"><a href="#" class="navigation__link">Планшеты</a></li>
+            <li class="navigation__item"><a href="#" class="navigation__link">Фото</a></li>
+            <li class="navigation__item"><a href="#" class="navigation__link">Видео</a></li>
+          </ul>
+        </nav>
+      </div>
+    </header>
+    <main class="cards">
+      <div class="card" data-qa="card">
+        <div class="card__item">
+          <div class="card__img">
+            <img
+              class="card__picture"
+              src="images/imac.jpeg"
+              alt="item picture">
+          </div>
+          <div class="card__name">Моноблок APPLE A1419 iMac 27" Retina 5K (MNED2UA/A)</div>
+          <div class="card__code card-font">Код товара: 195434</div>
+          <div class="card__about">
+            <div class="stars card__font">
```

Comment:
```suggestion
            <div class="stars card__font stars--4">
```
Должно работать с модификатором, как в задании stars